src/blocks/separator/Separator.types.ts

```diff
@@ -0,0 +1,61 @@
+import { CSSProperties } from 'react';
+
+import { BlocksColors, ResponsiveProp, SpaceType, ValueOf } from '../Blocks.types';
+import { FlattenSimpleInterpolation } from 'styled-components';
+
+export type SeparatorResponsiveProps = {
+  height?: ResponsiveProp<string>;
+  /* Sets margin css property */
+  margin?: ResponsiveProp<SpaceType>;
+  /* Sets max-height css property */
+  maxHeight?: ResponsiveProp<string>;
+  /* Sets min-height css property */
+  minHeight?: ResponsiveProp<string>;
+  /* Sets max-width css property */
+  maxWidth?: ResponsiveProp<string>;
+  /* Sets min-width css property */
+  minWidth?: ResponsiveProp<string>;
+  /* Sets padding css property */
+  padding?: ResponsiveProp<SpaceType>;
+  /* Sets width css property */
+  width?: ResponsiveProp<string>;
+};
+
+export type SeparatorNonResponsiveProps = {
+  /* Sets border css property */
+  border?: string;
+  /* Sets border-radius css property */
+  borderRadius?: string;
+  /* Sets background-color css property */
+  backgroundColor?: BlocksColors;
+  /* Sets position css property */
+  position?: CSSProperties['position'];
+  /* Sets box-shadow css property */
+  boxShadow?: string;
```

Comment:
None of these props are required. Separator doesn't needs these css properties.

Reply:
It is possible that we might need to add a `borderRadius` and a `backgroundColor` to the separator right? Agree that the rest have an extremely rare use case.

Reply:
No we don't. The whole idea is to make is consistent and use a standard version. It should be same through out the app